.cirrus.yml

```diff
@@ -0,0 +1,12 @@
+task:
+  name: seedvault-aosp
+  timeout_in: 120m
+  container:
+    image: ubuntu:23.04
+    cpu: 8
+    memory: 32G
+  build_script:
+    - ./.github/scripts/build_aosp.sh aosp_arm64 eng android-14.0.0_r1
```

Comment:
Any reason to go with `eng` instead of `userdebug` or just `user` even?

For an app like this it shouldn't make too big of a runtime difference. I'd default to `userdebug`.

Reply:
Sure. Will add this to my other PR to avoid wasting build minutes.

Reply:
Could also just cancel the build right after push. :)